cmd/vic-machine/converter/converter.go

```diff
@@ -328,6 +329,19 @@ func convertIP(src reflect.Value, prefix string, tags reflect.StructTag, dest ma
 	return nil
 }
 
+func convertTrustLevel(src reflect.Value, prefix string, tags reflect.StructTag, dest map[string][]string) error {
+	log.Debugf("convertTrustLevel: prefix: %s, src: %s", prefix, src.String())
+	trustLevel, ok := src.Interface().(common.TrustLevel)
+	if !ok {
+		panic(fmt.Sprintf(src.Type().String() + " is not TrustLevel"))
```

Comment:
Realistically, `ConvertTrustLevel` is only called when the type of a struct member is of type `common.TrustLevel`, so the panic should never be reached in any scenario. If there was some issue, the panic will help to triage the code error similar to the surrounding convert functions.